libs/SalesforceSDKCore/SalesforceSDKCore/Classes/SFLoginViewController.h

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
The classes of SalesforceSDKCore are generally further sub-categorized on the filesystem.  And while these have been in Project Navigator, they're all physically dumped under Classes/ on the filesystem.

Those should be moved to the physical counterparts of their logical folder structure.